app/brave_command_ids.h

```diff
@@ -128,6 +128,8 @@
 #define IDC_TILE_TABS 56326
 #define IDC_BREAK_TILE 56327
 #define IDC_SWAP_SPLIT_VIEW 56328
+#define IDC_TOGGLE_SPLIT_VIEW_ORIENTATION 56329
```

Comment:
This line seems to cause build failure. We should remove it.

Reply:
Done @sangwoo108